math/krishnamurthy_test.go

```diff
@@ -0,0 +1,51 @@
+package math
+
+import (
+	"fmt"
+	"testing"
+)
+
+func retCases() []struct {
+	input        int64
+	output       bool
+	outputString string
+} {
+	return []struct {
+		input        int64
+		output       bool
+		outputString string
+	}{
+		{-3112312321, false, "is not"},
+		{0, false, "is not"},
+		{1, true, "is"},
+		{2, true, "is"},
+		{109, false, "is not"},
+		{145, true, "is"},
+		{943, false, "is not"},
+		{6327, false, "is not"},
+		{40585, true, "is"},
+		{9743821, false, "is not"},
+		{3421488712, false, "is not"},
+	}
+}
+
+func TestIsKrishnamurthyNumber(t *testing.T) {
+	for _, test := range retCases() {
+		t.Run(fmt.Sprintf("%d ", test.input)+test.outputString+" a Krishnamurthy Number", func(t *testing.T) {
```

Comment:
You're already using the `Sprintf`, please use the verbs instead of string concatenating, this causes confusion to the reader.

Reply:
Hi Taj, please review the latest changes.